js/data/data_source.d.ts

```diff
@@ -236,15 +238,15 @@ export default class DataSource {
      * @return object
      * @public
      */
-    loadOptions(): any;
+    loadOptions(): {sort?: SortDescriptor | Array<SortDescriptor>, filter?: any, select?: SelectDescriptor, group?: GroupDescriptor | Array<GroupDescriptor>, requireTotalCount?: boolean};
```

Comment:
FilterDescriptor is missing:

```suggestion
    loadOptions(): {sort?: SortDescriptor | Array<SortDescriptor>, filter?: any, select?: SelectDescriptor, group?: GroupDescriptor | Array<GroupDescriptor>, requireTotalCount?: boolean};
```

Reply:
Check if `LoadOptions` type can be used here